src/ralph/admin/templates/admin/change_list_results.html

```diff
@@ -56,4 +56,11 @@
       </tbody>
     </table>
   </div>
+{% else %}
+  <div data-alert class="alert-box info">
+    {% url cl.opts|admin_urlname:'add' as add_url %}
+    {% blocktrans with cl.opts.verbose_name as name %}
+      Nothing to display. You can <a href="{{ add_url }}">add first {{ name }}</a>.
```

Comment:
I'm not sure if "first" is the best word here - what if empty list is result of filtering/searching?